client/components/PetDetailShow.js

```diff
@@ -1,9 +1,14 @@
 import React, { useState, useEffect } from "react"
 
-const PetDetailShow = (props) => {
+import PetAdoptionForm from "./PetAdoptionForm"
 
+const PetDetailShow = (props) => {
   const [showErrors, setShowErrors] = useState(false)
 
+  const [formVisibility, setFormVisibility] = useState("is-hidden")
+
+  const [messageVisibility, setMessageVisibility] = useState("is-hidden")
+

```

Comment:
It's ok to have the `useState` statements grouped with one empty line at the end like:

```js
  const [showErrors, setShowErrors] = useState(false)
  const [formVisibility, setFormVisibility] = useState("is-hidden")
  const [messageVisibility, setMessageVisibility] = useState("is-hidden")
  
```